e2e-tests/trade-finance-manager/cypress/support/reference-data-api/api.js

```diff
@@ -1,5 +1,5 @@
 // mock out number generator call and use 10 digit ID instead
-let ukefId = 1e10;
+let ukefId = 1e7;
```

Comment:
Can you please write a comment on why this was reduced.
Hint : FacilityURN for premium schedule segments cannot handle large numbers.

Reply:
done

Reply:
I'm confused here. because normally, ukefDealId has 10 characters. 
Are you getting any specific errors for this?

Reply:
just a 404

Reply:
and what are you getting if you have 7 chars?